MICS6_Master.do

```diff
@@ -34,16 +36,20 @@ macro drop _all
 	if `pc' == 0 global OUT "${root}/FINAL"
 	if `pc' == 1 global OUT "${root}/STATA/DATA/SC/ADePT READY/MICS/New"
 	if `pc' == 2 global OUT "${root}/STATA/DATA/SC/FINAL"
+	if `pc' == 3 global OUT "${root}/etc/output"
+	
 
 * Define path for INTERMEDIATE
 	if `pc' == 0 global INTER "${root}/INTER"
 	if `pc' == 1 global INTER "${root}/STATA/DATA/SC/ADePT READY/MICS/Inter"
 	if `pc' == 2 global INTER "${root}/STATA/DATA/SC/INTER"
+	if `pc' == 3 global INTER "${root}/etc/inter"
 
 * Define path for do-files
 	if `pc' == 0 global DO "/Users/zetianyuwang/Documents/PT_Data Whale/HEFPI/Code_github/MICS6"
 	if `pc' == 1 global DO "${root}/STATA/DO/SC/06_Prepare_MICS6/MICS6_DW"
 	if `pc' == 2 global DO "${root}/MICS6"
+	if `pc' == 3 global DO "D:/Drives/Github_Ortsang/MICS6"
 
 * Define the country names (in globals) by recode version
 	global newMICS6countries "Thailand2019"
```

Comment:
Just want to make sure that you have changed Thailand2019 to Tonga in your own version?